crtauth/protocol.py

```diff
@@ -165,7 +166,7 @@ class VerifiablePayload(SerializablePacket):
     payload - The payload.
     """
 
-    __magic__ = 'v'
+    __magic__ = six.b('v')
```

Comment:
read [here](https://pythonhosted.org/six/index.html#six.b) that the function is just for compatibility with python < 2.6 which I don't think we care about and that it's equivalent with b prefixed string literals in both python 2 and 3.